src/015/p015.clj

```diff
@@ -0,0 +1,15 @@
+(def init [1 1])
+
+(defn step [line] 
+  (->> (concat [0] line [0])
+       (partition 2 1)
+       (mapv (partial apply +))))
+
+(defn solve [] 
+  (->> init
+      (iterate step)
+      (drop 19)
+      first
+      (reduce ((map #(* % %)) +))))
+
+(prn (solve))
```

Comment:
Did you originally intend to add a Clojure solution here as well?
(Also, missing newline 😅).

Reply:
not really...